### PR TITLE
Update kr.js

### PR DESCRIPTION
**Pull request by Baw-Appie**

### CODE REVIEW

**Comment by mty22** [comment]
Thank you!

**Comment by goonmandu** [comment]
There are many incorrect/vague/unnatural translations. I suggest we base this update's translations on my work.
@zVaderX and I have pretty much the same translations, with mine being actually in the code while Vader's is a list of comments.
Below are some of the problems I found with this PR:
```diff
+ 361: settings.editing.ropecol 그래플 줄 색상 (there are plenty of Korean translations for "grappling hook" such as 갈고리)
- 470: windows.controls.kickVoteY 추방 투표 [네] (no context given for broad term "추방")
+ 471: windows.controls.kickVoteN 추방 투표 [아니요] (same problem as above)
- 475: windows.controls.specObj 목표 카메라 전환 (inadequate direct translation of "objective," a better one would be "거점")
+ 476: windows.controls.specFirst 첫 사람 전환 (inadequate direct translation of "first person," the correct TL would be "1인칭")
- 690: server.config.fallDmgThr 추락 데미지 뒹굴기 (wrong translation of "threshold," considering this decides the height at which a player takes fall damage, it would be better to just say "높이 (height)")
```

**Comment by Baw-Appie** [comment]
"그래플" was translated based on the pronunciation of "스나이퍼" such as 147 lines, but "갈고리" also seems to be fine.
"추방" was translated based on line 782 "windows.clan.kick {0}님을 클랜에서 추방하시겠습니까?"

Other wrong translation are corrected.

**Comment by zVaderX** [comment]
94 히트레그 테스트
180 거래 불가능 한 스킨 숨기기
265 게임플레이 설정
305 뷰 모델 설정
361 그래플 줄 색깔
470 추방 투표 [예]
471 추방 투표 [아니요]
472 KPD 투표 [예]
473 KPD 투표 [아니요]
475 목표 카메라 전환
476 첫 사람 전환 or 1인칭 전환
487 개발자 툴 1
488 개발자 툴 2
489개발자 툴 3
490 개발자 툴 4
581 커스텀 게임만 가능
582 프리미엄만 가능
676 휴식 시간
689 낙하 데미지
690 추락 데미지 뒹굴기 or 추락 데미지 높이
691 무승부 불가능
692 가장 좋은
784 {0} 에게 {1} 일의 프리미엄을 선물하시겠습니까?

So this is the final translations?

File: kr.js [hunk 1]
```diff
@@ -1,821 +1,821 @@
...
+settings.interface.critCol 치명타 색상
+settings.interface.header 인터페이스
+settings.interface.scale 인터페이스 크기
+settings.interface.ui 인터페이스 표시
+settings.interface.scoreboard 구 점수판 사용
+settings.interface.oldBrowser 구 서버 탐색기 사용
+settings.interface.rarityanim 희귀도 애니메이션 끄기
+settings.interface.showPfp 프로필 사진 표시
+settings.interface.showTradeMsg 트레이드 메시지 표시
+settings.interface.hideNonTrade 거래 불가능
```
Comment (YuniQ152) [inline kr.js:180]: Line 180: 거래 불가 스킨 숨기기

File: kr.js [hunk 2]
```diff
@@ -1,821 +1,821 @@
...
+settings.editing.saturation 그래픽 채도
+settings.editing.menuSaturation 인터페이스 채도
+settings.editing.hue 그래픽 색상
+settings.editing.menuHue 인터페이스 색상
+settings.editing.vignette 화면 테두리 어둡게 하기
+settings.editing.feedLimit 킬로그 제한
+settings.editing.depth 깊이 지도
+settings.editing.green 크로마 키
+settings.editing.trailcol 총알 흔적 색상
+settings.editing.ropecol 그래플 줄 색상
```
Comment (YuniQ152) [inline kr.js:361]: Line 361: 갈고리 줄 색상

File: kr.js [hunk 3]
```diff
@@ -1,821 +1,821 @@
...
+windows.controls.confirm 사용하기 확인
+windows.controls.drop 무기 떨어뜨리기
+windows.controls.wepVis 무기 보이기 전환
+windows.controls.kickVoteY 추방 투표 [네]
+windows.controls.kickVoteN 추방 투표 [아니요]
+windows.controls.kpdVoteY KPD 투표 [네]
+windows.controls.kpdVoteN KPD 투표 [아니요]
+windows.controls.specFree 자유 카메라 전환
+windows.controls.specObj 거점 카메라 전환
+windows.controls.specFirst 1인칭 전환
```
Comment (YuniQ152) [inline kr.js:476]: Line 474: 자유 시점 전환
Line 475: 거점 시점 전환
Line 476: 1인칭 시점 전환

File: kr.js [hunk 4]
```diff
@@ -1,821 +1,821 @@
...
+server.config.kCams 처치 영상
+server.config.noReload 재장전 없음
+server.config.private 비공개
+server.config.krRewards KR 보상
+server.config.forceDefault 기본 설정 강제
+server.config.warmupTime 준비 시간
+server.config.gamRounds 라운드
+server.config.objtvTime 거점 변경 시간
+server.config.forceSpawn 강제 리스폰 시간
+server.config.intermTmr 중지 시간 (초)
```
Comment (YuniQ152) [inline kr.js:676]: Line 676: 휴식 시간 (초)

File: kr.js [hunk 5]
```diff
@@ -1,821 +1,821 @@
...
+server.config.hpRegen HP 재생
+server.config.disableB 경계 비활성화
+server.config.throwMel 근접무기 던지기
+server.config.noCosm 외형 아이템 비활성화
+server.config.tstCmp 테스트 모드
+server.config.limitClasses 클래스 제한
+server.config.tmSize 팀 크기
+server.config.fallDmg 추락 데미지
+server.config.fallDmgThr 추락 데미지 높이
+server.config.noDraws 무승부
```
Comment (YuniQ152) [inline kr.js:691]: Line 691: 무승부 불가

File: kr.js [hunk 6]
```diff
@@ -1,821 +1,821 @@
...
 server.config.bstOfR Best of
```
Comment (YuniQ152) [inline kr.js:692]: Line 692: 다전제

File: kr.js [hunk 7]
```diff
@@ -1,821 +1,821 @@
...
+windows.store.currency * 미국 달러(USD)로 표시된 가격은 결제 제공업체나 은행에서 정한 환율에 따라 현지 통화로 환전될 수 있습니다
+windows.store.creator 당신이 가장 좋아하는 크리에이터를 위해 크리에이터 코드를 사용하세요
+windows.chall.customs 사용자 지정 게임에서는 챌린지를 진행할 수 없습니다
+twitch.link Twitch 계정을 연동하겠습니까?
+twitch.unlink Twitch 계정의 연동을 해제하시겠습니까?
+windows.clan.leave 클랜을 탈퇴하시겠습니까?
+windows.clan.delete 클랜을 해체하시겠습니까?
+windows.clan.kick {0}님을 클랜에서 추방하시겠습니까?
+windows.store.premium.purchase {0}일 치의 프리미엄 서비스를 구매하시겠습니까?
+windows.store.premium.gift '{0}'에게 {1} 일의 Premium을 선물할까요?
```
Comment (YuniQ152) [inline kr.js:784]: Line 784: '{0}'님에게 {1}일 치의 프리미엄을 선물하시겠습니까?

File: kr.js [hunk 8]
```diff
@@ -1,821 +1,821 @@
...
+windows.publish.mod.header 모드 업로드
+windows.publish.mod.thumbnail 썸네일 없음
+windows.publish.mod.name 모드 이름
+windows.publish.mod.paste 모드 URL
+windows.publish.mod.update 업로드
+windows.publish.mod.login 해서 모드를 만들고 업로드하세요!
+windows.secondary.header 보조무기 선택
+windows.secondary.unlocked 잠금 해제됨
+windows.secondary.customOnly 커스텀 전용
+windows.secondary.premiumOnly Premium 전용
```
Comment (YuniQ152) [inline kr.js:582]: Line 582: 프리미엄 전용

File: kr.js [hunk 9]
```diff
@@ -1,821 +1,821 @@
...
+windows.controls.board 리더보드
+windows.controls.interact 주 사용하기
+windows.controls.interactSec 부 사용하기
+windows.controls.confirm 사용하기 확인
+windows.controls.drop 무기 떨어뜨리기
+windows.controls.wepVis 무기 보이기 전환
+windows.controls.kickVoteY 추방 투표 [네]
+windows.controls.kickVoteN 추방 투표 [아니요]
+windows.controls.kpdVoteY KPD 투표 [네]
+windows.controls.kpdVoteN KPD 투표 [아니요]
```
Comment (YuniQ152) [inline kr.js:473]: Line 470: 추방 투표 [찬성]
Line 471: 추방 투표 [반대]
Line 472: KPD 투표 [찬성]
Line 473: KPD 투표 [반대]

**Review by YuniQ152 — COMMENTED** [review]
수정이 필요해 보이는 번역에 리뷰를 추가했습니다. 본 PR에서 새롭게 추가된 번역을 반영하고, 반영 후 #213 에서 기존 번역을 수정하는 것이 좋아 보입니다.

**Comment by mty22** [comment]
> There are many incorrect/vague/unnatural translations. I suggest we base this update's translations on my work.
> @zVaderX and I have pretty much the same translations when it comes to the new stuff, with mine being actually in the code while Vader's is a list of comments. Mine also has updates to the existing translations to fit the updated original English text placement.
> Below are some of the problems I found with this PR:
> 
> ```diff
> + 361: settings.editing.ropecol 그래플 줄 색상 (there are plenty of Korean translations for "grappling hook" such as 갈고리)
> - 470: windows.controls.kickVoteY 추방 투표 [네] (no context given for broad term "추방")
> + 471: windows.controls.kickVoteN 추방 투표 [아니요] (same problem as above)
> - 475: windows.controls.specObj 목표 카메라 전환 (inadequate direct translation of "objective," a better one would be "거점")
> + 476: windows.controls.specFirst 첫 사람 전환 (inadequate direct translation of "first person," the correct TL would be "1인칭")
> - 690: server.config.fallDmgThr 추락 데미지 뒹굴기 (wrong translation of "threshold," considering this decides the height at which a player takes fall damage, it would be better to just say "높이 (height)")
> ```

OK, noted.

It looks like most of the work is being done in this PR already, rather than https://github.com/mty22/krunker.io-translations/pull/213

We'll compensate KR to all parties involved accordingly, based on the changes made and requested.

@goonmandu I consider you the owner of `kr.js` file, and I trust @YuniQ152 due to him being consistent in the past with translations and reviews.

The next round of updates to `en.js`, @goonmandu you can have more control over the lifecycle of the changes to `kr.js`, and I'll send out the KR to people involved.

**Comment by mty22** [comment]
@goonmandu @YuniQ152 let me know when good to merge. :)

**Comment by goonmandu** [comment]
@Baw-Appie @mty22 @YuniQ152 
Sorry for the delayed feedback; I was busy with the upcoming college and immigration to the US. Since I will be going over the next patch (as previously mentioned in this pull request), I'll implement other small changes to the existing translations that the now-closed pull request of mine in the next update.

To begin, there is a crucial change suggested by @YuniQ152 (무승부 없음 on line 691) that has not been updated in this pull request.
```diff
+ 691: 무승부
- 691: The original English text is "No Draws," and therefore 
-      should be "무승부 없음" to correctly toggle this setting in-game.
```
There are some suggestions myself:
```diff
+ 111: 앵글 그래픽 백엔드
- 111: ANGLE is the WebGL graphics backend that Krunker uses, making it a proper noun.
-      Therefore, it should not be transliterated as "앵글" and rather be left alone as "ANGLE."

+ 180: 거래 불가 스킨 숨기기
- 180: The trading feature is now usually referred to as "트레이드" by the Korean playerbase
-      instead of the dictionary definition "거래"

+ 305: 모델 설정 보기
- 305: The translation key for this setting is "vmSetts." Judging by how it is camelCased, it is not supposed to 
-      let the player view the model settings, but to open the settings menu for the View Model page.

+ 784: '{0}'에게 {1} 일의 프리미엄을 선물할까요?
- 784: Remove the space between {1} and 일의, since, in Korean, it is more natural to write 
-      "30일" instead of "30 일," with a space separating the number from the text.
 ```
As soon as these changes are made, I say we are good to merge.

**Comment by mty22** [comment]
Let me know when ready to merge! :)

**Comment by goonmandu** [comment]
@mty22 Good to merge!

**Comment by mty22** [comment]
kr queued for all parties involved.

in future, please work together rather than create 3 separate PR's :)

**Pull request merged**